src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -35,68 +35,55 @@ static Certificates()
 
-            public static X509Certificate2 GetClientCertificate() => GetCertWithPrivateKey(GetClientCertificateCollection());
+            public static X509Certificate2 GetClientCertificate() => GetCertificate("testclienteku.contoso.com.pfx");
 
-            public static X509Certificate2 GetNoEKUCertificate() => GetCertWithPrivateKey(GetNoEKUCertificateCollection());
+            public static X509Certificate2 GetNoEKUCertificate() => GetCertificate("testnoeku.contoso.com.pfx");
 
-            public static X509Certificate2 GetSelfSignedServerCertificate() => GetCertWithPrivateKey(GetSelfSignedServerCertificateCollection());
+            public static X509Certificate2 GetSelfSignedServerCertificate() => GetCertificate("testselfsignedservereku.contoso.com.pfx");
 
-            public static X509Certificate2 GetSelfSignedClientCertificate() => GetCertWithPrivateKey(GetSelfSignedClientCertificateCollection());
+            public static X509Certificate2 GetSelfSignedClientCertificate() => GetCertificate("testselfsignedclienteku.contoso.com.pfx");
 
-            public static X509Certificate2Collection GetServerCertificateCollection() => GetCertificateCollection("testservereku.contoso.com.pfx");
-
-            public static X509Certificate2Collection GetClientCertificateCollection() => GetCertificateCollection("testclienteku.contoso.com.pfx");
+            public static X509Certificate2Collection GetServerCertificateCollection()
+            {
+                var certs = new X509Certificate2Collection();
+                certs.Add(GetServerCertificate());
 
-            public static X509Certificate2Collection GetNoEKUCertificateCollection() => GetCertificateCollection("testnoeku.contoso.com.pfx");
+                return certs;
+            }
 
-            public static X509Certificate2Collection GetSelfSignedServerCertificateCollection() => GetCertificateCollection("testselfsignedservereku.contoso.com.pfx");
+            public static X509Certificate2Collection GetClientCertificateCollection()
+            {
+                var certs = new X509Certificate2Collection();
+                certs.Add(GetClientCertificate());
 
-            public static X509Certificate2Collection GetSelfSignedClientCertificateCollection() => GetCertificateCollection("testselfsignedclienteku.contoso.com.pfx");
+                return certs;
+            }
 
-            private static X509Certificate2Collection GetCertificateCollection(string certificateFileName)
+            private static X509Certificate2 GetCertificate(string certificateFileName)
             {
                 // On Windows, .NET Core applications should not import PFX files in parallel to avoid a known system-level race condition.
                 // This bug results in corrupting the X509Certificate2 certificate state.
                 Assert.True(m.WaitOne(MutexTimeout), "Cannot acquire the global certificate mutex.");
```

Comment:
Now that you've updated the certs, does this race condition still exist, or can we remove this mutex?

Reply:
The race condition on the import of the PFX seems to be a different problem than the one I solved by updating the certs (to remove the friendlyname of private key).  I'm still investigating to see if we can remove the mutex.

Reply:
Update:  turns out I hadn't retested removing the Mutex since my very latest changes here. I'm able to remove it now and the tests pass.